src/BenchmarksDriver/Program.cs

```diff
@@ -411,6 +421,7 @@ public static int Main(string[] args)
                            ,[Scheme]
                            ,[ConnectionFilter]
                            ,[WebHost]
+                           ,[ThreadCount]
```

Comment:
Which old data?  The new column `ThreadCount` allowed null, so all existing rows have null.  If I rename the column to `KestrelThreadCount`, existing data in the column will be preserved.